### PR TITLE
Use jq to grab version from manifest.json

### PR DESCRIPTION
**Pull request by taymoork2**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -309,10 +300,11 @@ jobs:
       - checkout
       - run: git remote add upstream git@github.com:webex/react-widgets.git
       - run: git fetch --all --tags --prune
+      - jq/install
       - run:
           name: 'Grab Version Number'
           command: |
-            PRODUCTION_VERSION_NUMBER=$(curl -f 'https://code.s4d.io/widget-demo/production/.version')
+            PRODUCTION_VERSION_NUMBER=$(curl -s -f 'https://code.s4d.io/widget-space/production/manifest.json' | jq -r '.version')
```
Comment (adamweeks) [inline .circleci/config.yml]:
```suggestion
            PRODUCTION_VERSION_NUMBER=$(curl -s -f 'https://code.s4d.io/widget-space/production/manifest.json' | jq -r 'manifest.json')
```

**Comment by adamweeks** [comment]
Could you create a test PR that shows the circle job properly fetching and outputting the version to prove that these changes work?

Please link to the circle output here for that test PR for convenience.

File: .circleci/config.yml (hunk 1 above)
Comment (taymoork2) [reply .circleci/config.yml]: `.version` is jq syntax to grab the `version` key

**Comment by taymoork2** [comment]
https://circleci.com/gh/webex/react-widgets/10374

File: .circleci/config.yml (hunk 1 above)
Comment (adamweeks) [reply .circleci/config.yml]: Ah ok!

**Pull request merged**